ddtrace/internal/datadog/profiling/src/exporter.cpp

```diff
@@ -0,0 +1,560 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0. This product includes software
+// developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present
+// Datadog, Inc.
+
+// High-level skip for invalid architectures
+#ifndef __linux__
+#elif __aarch64__
+#elif __i386__
+#else
```

Comment:
This is probably to scope out some of the testing and build type changes.
The code seems to be OK for all these platforms / archs. 
I'm fine with leaving this. I'm just trying to figure out where the triggers should be.

Reply:
The immediate code should build, but libdatadog.a is a link-time requirement.

This is mostly just to spare me from having to diagnose any issues, should they arise, on build system configurations I don't care about yet (e.g., I can't control CI compiler versions at the moment).